code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -625,3 +617,38 @@
 		return
 	..()
 
+/datum/reagent/toxin/tobacco
+	name = "Space Tobacco"
+	id = "tobacco"
+	description = "Low-grade space tobacco."
+	reagent_state = LIQUID
+	color = "#333300"
+	data = 0
+	taste_description = "low-grade tobacco"
+	strength = 0.004
+	taste_mult = 10
+
+/datum/reagent/toxin/tobacco/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	var/mob/living/carbon/human/NewM = M
```

Comment:
This is dumb. Just declare the argument as carbon/human and do `if (!istype(H)) return`.